### PR TITLE
docs(README): added unit conversion example

### PR DESCRIPTION
**Pull request by igiona**
Firstmost: thank you for the great work here 👏

This PR contains really a little change in the README, but this fundamental thing was hard to find for me.
I did not really find this hint in the documentation or I was not able to simply get hold of it.

With this simple example, I hope to spare someone's time 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,9 @@ fn main() {
     let velocity/*: Velocity*/ = length / time;
     let _acceleration = calc_acceleration(velocity, time);
     //let error = length + time; // error[E0308]: mismatched types
+
+    // Get a quantity value in a specific unit
```
Comment (iliekturtles) [inline README.md:56]:
```suggestion
    // Get a quantity value in a specific unit.
```

**Comment by iliekturtles** [comment]
Could you squash the commits and I'll merge.

**Comment by igiona** [comment]
@iliekturtles it's done 😉

**Comment by igiona** [comment]
@iliekturtles I'm closing this one because it keeps showing up in my PR list.

Feel free to re-open and merged whenever it suite you best.

**Comment by iliekturtles** [comment]
Sorry for the extended delay! Merged!

**Pull request merged**